src/wui/attack_window.cc

```diff
@@ -123,6 +123,8 @@ AttackPanel::AttackPanel(
      target_coordinates_(target_coordinates),
      get_max_attackers_(get_max_attackers),
      attack_type_(attack_type),
+     icon_w_(attack_type_ == AttackPanel::AttackType::kShip ? 64 : 32),
```

Comment:
This should depend on `kHealthBarWidth` from the ship code, so it should be moved to `ship.h`.